include/sqlpp11/aggregate_functions/trim.h

```diff
@@ -0,0 +1,110 @@
+/*

```

Comment:
This file should be located in `sqlpp11`, not in `sqlpp11/aggregate_functions` (I just pointed you there for examples, sorry for the confusion).